Tests/FuzzilliTests/AnalyzerTest.swift

```diff
@@ -126,7 +126,9 @@ class AnalyzerTests: XCTestCase {
             cls.defineConstructor(withParameters: [.string]) { _ in
                 XCTAssertEqual(b.context, [.script, .classDefinition, .function])
                 let v0 = b.loadInt(42)
-                let v1 = b.createObject(with: ["foo": v0])
+                let v1 = b.createObject { prop in
```

Comment:
Let's unify the closure argument (`prop`) in this PR. How about just `obj` (if we also support defining methods at least)?
```swift
let v1 = b.createObject { obj in
    obj.addProperty("foo", withValue: v0)
}
```